yt/visualization/volume_rendering/scene.py

```diff
@@ -349,8 +349,10 @@ def save(
             ax = fig.add_axes([0, 0, 1, 1])
             ax.set_axis_off()
             out = self._last_render
-            nz = out[:, :, :3][out[:, :, :3].nonzero()]
-            max_val = nz.mean() + sigma_clip * nz.std()
+            if sigma_clip:
```

Comment:
better avoid treating 0 as None (sigma_clip=0 means « clip to the mean value », right ?)
```suggestion
            if sigma_clip is not None:
```